_episodes/02-running-containers.md

```diff
@@ -180,12 +180,29 @@ with Singularity available.
 > {: .solution}
 {: .challenge}
 
-## Exiting a singularity image
+## Exiting a Singularity image
 
-The `exit` command exits a singularity instance. Note that when exiting from the singularity image all the running processes are killed (stopped).
+The `exit` command exits a Singularity instance. Note that when exiting from the Singularity image all the running processes are killed (stopped).
```

Comment:
Can you rephrase: Note that all running processes ... when exiting from the ....? (clearer that way)

Reply:
Done